README.md

```diff
@@ -4,33 +4,9 @@
 
 The documentation repository contains the documentation for OpenSearch, the search, analytics, and visualization suite with advanced security, alerting, SQL support, automated index management, deep performance analysis, and more. You can find the rendered documentation at [opensearch.org/docs](https://opensearch.org/docs).
 
-
-## How you can help
-
-Community contributions remain essential in keeping this documentation comprehensive, useful, well-organized, and up-to-date. If you are interested in contributing, please see the [Contribution](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file.
-
-
-## Points of contact
-
-If you encounter problems or have questions when contributing to the documentation, these people can help:
-
-- [cwillum](https://github.com/cwillum)
-- [hdhalter](https://github.com/hdhalter)
-- [kolchfa-aws](https://github.com/kolchfa-aws)
-- [Naarcha-AWS](https://github.com/Naarcha-AWS)
-- [vagimeli](https://github.com/vagimeli)
-
-
 ## How the website works
 
-This repository contains [Markdown](https://guides.github.com/features/mastering-markdown/) files organized into Jekyll "collections" (e.g., `_search-plugins`, `_opensearch`, etc.). Each Markdown file correlates with one page on the website.
-
-Using plain text on GitHub has many advantages:
-
-- Everything is free, open source, and works on every operating system. Use your favorite text editor, Ruby, Jekyll, and Git.
-- Markdown is easy to learn and looks good in side-by-side diffs.
-- The workflow is no different than contributing code. Make your changes, build locally to check your work, and submit a pull request. Reviewers check the PR before merging.
-- Alternatives like wikis and WordPress are full web applications that require databases and ongoing maintenance. They also have inferior versioning and content review processes compared to Git. Static websites, such as the ones Jekyll produces, are faster, more secure, and more stable.
+This repository contains [Markdown](https://guides.github.com/features/mastering-markdown/) files organized into Jekyll _collections_ (for example, `_api-reference` or `_dashboards`). Each Markdown file corresponds to one page on the website.
```

Comment:
Do we need this section here? Would it be more appropriate in the contribution file?

Reply:
The readme is the what and the contributing is the how in my opinion. So readme is about this repo and contributing is about how to contribute. So I think it's better to have it here.